core/templates/dev/head/app.constants.ts

```diff
@@ -0,0 +1,163 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Shared constants for the Oppia module.
+ */
+
+oppia.constant('RULE_SUMMARY_WRAP_CHARACTER_COUNT', 30);
```

Comment:
Just to confirm ... you've verified that every single constant in this file is used in > 1 file? If not, please do.

Reply:
@seanlip I previously followed the way that all the constants in App.ts are to be in the app level file. But there were some which were not used in multiple files. So I have put them in their respective page constants file. PTAL!

Reply:
Also there were some constants used in single file but they were from the services/ folder which is a shared directory and hence such constants are still in this app level file.

Reply:
I don't mind as long as we have a clear and consistent convention that developers can easily understand.

Reply:
I also suggest that you include, with this PR, a means of enforcing this convention through the lint checks.

Reply:
@seanlip I have added the check! PTAL!

Reply:
It's a good check, but I think you also need a check to ensure that each constant is declared exactly once globally.

Reply:
@YashJipkate Please respond to all inline comments.

Reply:
Sorry for not responding. But this comment is already addressed. I have incorporated a lint check to ensure no repetition of constant declaration.